### PR TITLE
Review people filter

### PR DESCRIPTION
**Pull request by janwerkhoven**
In this PR the people filter is being reviewed as per request of Nina at Red Dust.

* [x] People should have only 1 category (board, ambassador, role model, staff, ...)
* [x] People should sort according their category
* [x] Rename to filter, as the wording "sorting" is not correct

### CODE REVIEW

File: src/js/main.js [hunk 1]
```diff
@@ -142,8 +142,59 @@ $(document).ready(function() {
...
+      });
+
+      // 2. Sort each individual alphabetically by their name
+      var sortedPeopleInCategory = peopleInCategory.sort(function(a, b) {
+        if (a.name < b.name)
+          return -1;
+        if (a.name > b.name)
+          return 1;
+        return 0;
+      });
```
Comment (richardverheyen) [inline src/js/main.js:179]: you surprise me, I thought you might put this in one line ;)

**Review by richardverheyen — COMMENTED** [review]
Yeah cool, it all looks pretty good, I'll quickly pull it down locally and test it before approving

File: src/js/main.js (hunk 1 above)
Comment (janwerkhoven) [reply src/js/main.js:179]: Hehe I admit, I copied this one from Stack Overflow.

Comment (janwerkhoven) [reply src/js/main.js:179]: Here you go Sir `return (a > b) ? 1 : ((a < b) ? -1 : 0);` ;)

**Pull request merged**